### DISCUSSION TITLE
能简单说明一下您需要宽字符串的场景吗

### DISCUSSION THREAD

**Discussion by wszqkzqk** [dbkangaroo/kangaroo#112] [Q&A]
Vala其实比较倾向于什么都用UTF-8来做，感觉对宽字符串的函数支持和语言集成都不完善，您是在 什么场景下需要这些功能。以及在这里讨论可能不太方便，有没有其他的联系方式。

**Comment by wszqkzqk** [comment]
@taozuhong 还有一点，Vala现在只有unichar (32-bit)和unichar2 (16-bit)，没有wchar_t

**Reply by taozuhong** [reply]
Vala 语言爱好者QQ群：721531015
